setup.py

```diff
@@ -40,16 +40,16 @@ def _install_requirements():
     return requirements
 
 setup(
-    name='atomicapp',
-    version='0.1.9',
```

Comment:
wouldn't it be easier to have a file in the root folder named VERSION that you can read it from? seems that whenever a new version comes out you have to s/0.1.9/0.2.0/ every file